apps/full-site-editing/package.json

```diff
@@ -43,6 +43,6 @@
 		"@wordpress/plugins": "^2.2.0",
 		"@wordpress/url": "^2.5.0",
 		"classnames": "^2.2.6",
-		"lodash": "^4.17.11"
+		"lodash": "4.17.12"
```

Comment:
This gets externalized by the wordpress external dependencies plugin, so it doesn't matter what version it says here in package.json

https://github.com/Automattic/wp-calypso/blob/c10639fb201174b382c5b69349f72939f34738d8/packages/wordpress-external-dependencies-plugin/index.js#L20